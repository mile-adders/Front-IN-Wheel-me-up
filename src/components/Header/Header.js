/* eslint-disable no-unused-vars */
import React from 'react';
import {Link, NavLink} from 'react-router-dom';
import Hamburger from './hamburger/hamburger.js';
import NavBar from '../navBar/navBar.js'
export default function Header (){
  return(
    <header data-aos='fade-down'>
      {/* <nav className="nav-wrapper red"> */}
        <div className="container">
          <a href="/" className="brand-logo">
            <img className='logo' src={require('./assets/drive.png')}></img>
            <div className='appName'>Wheel Me Up</div>
          </a>
        </div>
        {/* <Hamburger /> */}
        <NavBar />
      {/* </nav> */}
    </header>
  );
}