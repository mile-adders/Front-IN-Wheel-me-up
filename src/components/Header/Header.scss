header{
    padding-top: 5px;
    padding-bottom: 5px;
      width: 100%;
      background-color: black;
      margin: 0 auto;
      margin-bottom: 1vw;
      h1{
          color: white;
          margin-left: 3vw;
      }
    //   ul{
    //       list-style-type: none;
    //   }
    //   li {
    //     color: white;
    //     font-family: $headerFont;
    //     font-weight: bold;
    //     float: right;
    //     margin-left: 2vw;
    //     margin-right: 2vw;
    //     margin-top: 1vw;
    //     text-decoration: none;
    //   }
    }