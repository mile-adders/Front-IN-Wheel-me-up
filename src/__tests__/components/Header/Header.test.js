/* eslint-disable no-unused-vars */
import React from 'react';
import {shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';
import Header from '../../../components/Header/Header.js';

describe('<Header />', ()=>{
  // it('exists at the start of our application',()=>{
  //   let app = shallow(<Header />);
  //   expect(app.find('h1').exists()).toBeTruthy();
  //   expect(app.find('h1').text()).toContain('Wheel Me Up');
  // });
  it('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});