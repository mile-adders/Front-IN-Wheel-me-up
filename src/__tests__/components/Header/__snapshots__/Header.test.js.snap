// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<header>
  <h1>
    Wheel Me Up
  </h1>
</header>
`;
